plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -1827,6 +1827,21 @@ public final class CoreMetrics {
     .setHidden(true)
     .create();
 
+  /**
+   * @since 10.4
+   *
+   * This metric is only needed to present additional information for pull requests about number of issues that pull request might be
+   * fixing.
+   */
+  public static final String PULL_REQUEST_FIXED_ISSUES_KEY = "pull_request_fixed_issues";
+  public static final Metric<Metric.Level> PULL_REQUEST_FIXED_ISSUES = new Metric.Builder(PULL_REQUEST_FIXED_ISSUES_KEY,
+    "Pull request fixed issues", Metric.ValueType.LEVEL)
```

Comment:
The value type should be int, like the other issue counts

Reply:
Thank you for your comment. Bad copy paste.